build.sbt

```diff
@@ -67,7 +67,7 @@ lazy val common = (project in file("common")).
       "org.json4s"   %% "json4s-jackson" % "3.3.0",
       "org.slf4j" % "slf4j-api" % "1.7.13",
       "org.slf4j" % "jul-to-slf4j" % "1.7.13",
-      "org.apache.kafka" % "kafka-clients" % "0.9.0.1",
+      "org.apache.kafka" % "kafka-clients" % "0.10.1.0",
```

Comment:
Shouldn't this be pulled in through a transitive dependency on the kafka-consumer lib?